WordPress/wordcamp.org
# Fonts: Add custom fonts for WordCamp Asia

Fixes #1189

Adopt the method outlined in issue #1216 as a temporary solution for loading custom fonts for WordCamp Asia. 
Additionally, removing the 'preview' properties does not affect functionality, so it has been removed as well. (Tested on sandbox)




### Screenshots

![image](https://github.com/WordPress/wordcamp.org/assets/18050944/bb3270ac-042d-4a6c-832b-30c2267b2957)
![image](https://github.com/WordPress/wordcamp.org/assets/18050944/52eb9667-0f78-4436-910c-4da0461f7b91)
![Screenshot 2024-01-26 at 03 09 21](https://github.com/WordPress/wordcamp.org/assets/18050944/9092844f-ddac-4f0c-8809-6691b2caa841)
![Screenshot 2024-01-26 at 03 09 32](https://github.com/WordPress/wordcamp.org/assets/18050944/252b9321-a348-481d-8c5c-8da19fb8c5c3)
![Screenshot 2024-01-26 at 03 09 39](https://github.com/WordPress/wordcamp.org/assets/18050944/156c3c9a-36a6-486d-9ae6-ca7a481cd292)



### How to test the changes in this Pull Request:

1. Add some text in the editor and use the new fonts, it should render the fonts in either the editor or the frontend.
